var/spack/repos/builtin/packages/py-bluepy/package.py

```diff
@@ -40,8 +40,10 @@ class PyBluepy(PythonPackage):
     depends_on('py-pylru@1.2:', type='run', when='@:2.2.9')
 
     # morph-tool
-    depends_on('py-morph-tool@2.4.0:', type='run', when='@2.3.0:')
-    depends_on('py-morphio@3.0.0:', type='run', when='@2.3.0:')
+    depends_on('py-morph-tool@2.4.3:2.999', type='run', when='@2.3.0:')
+
+    # morphio
```

Comment:
Could we in the future leave the obvious comments out?

Reply:
Sure, I think that the bluepy spec will be cleaned when the old versions will be removed.

Reply:
indeed. Not sure if a package still use version of bluepy<2.0.0.